main/main.cpp

```diff
@@ -2757,9 +2757,14 @@ Error Main::setup2(bool p_show_boot_logo) {
 								prefer_wayland_found = true;
 							}
 
-							if (!remember_window_size_and_position_found && assign == "interface/editor/remember_window_size_and_position") {
+							if (editor && !remember_window_size_and_position_found && assign == "interface/editor/editor_screen") {
```

Comment:
You could probably use the `screen_property` above.

Reply:
Refactored this a bit, realized that saving the position of the project manager was never implemented